src/lib/viewers/controls/media/PlayPauseToggle.tsx

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import noop from 'lodash/noop';
+import IconPlay24 from '../icons/IconPlay24';
+import IconPause24 from '../icons/IconPause24';
+import MediaToggle from './MediaToggle';
+import './PlayPauseToggle.scss';
+
+export type Props = {
+    isPlaying?: boolean;
+    onPlayPause: (isPlaying: boolean) => void;
+};
+
+export default function PlayPauseToggle({ isPlaying, onPlayPause = noop }: Props): JSX.Element {
+    const Icon = isPlaying ? IconPause24 : IconPlay24;
+    const title = isPlaying ? __('media_pause') : __('media_play');
+
+    return (
+        <MediaToggle className="bp-PlayPauseToggle" onClick={(): void => onPlayPause(!isPlaying)} title={title}>
```

Comment:
Is the return type on the `onClick` callback needed?

Reply:
Yeah, every function needs an explicit return type per our current TS config.